conntrack_linux.go

```diff
@@ -159,13 +159,15 @@ func (h *Handle) ConntrackDeleteFilter(table ConntrackTableType, family InetFami
 // ConntrackDeleteFilters deletes entries on the specified table matching any of the specified filters using the netlink handle passed
 // conntrack -D [table] parameters         Delete conntrack or expectation
 func (h *Handle) ConntrackDeleteFilters(table ConntrackTableType, family InetFamily, filters ...CustomConntrackFilter) (uint, error) {
+	var errMsgs []string
 	res, err := h.dumpConntrackTable(table, family)
-	if err != nil {
+	if err != nil && !errors.Is(err, ErrDumpInterrupted) {
 		return 0, err
 	}
 
+	errMsgs = append(errMsgs, err.Error())
```

Comment:
this allows us to at least do a best effort to try to clean the entries matching the filter

Reply:
Just occurred to me; err may be `nil` here? Should probably be inside the `err != nil` branch? Perhaps  also a comment couldn't hurt?

```go
if err != nil {
	if  !errors.Is(err, ErrDumpInterrupted)  {
		return 0, err
	}
	// This allows us to at least do a best effort to try to clean the
	// entries matching the filter.
    errMsgs = append(errMsgs, err.Error())
}
```

Reply:
Good catch, thank you. Fixed in 2742887